jerry-core/ecma/operations/ecma-bigint.c

```diff
@@ -147,7 +154,6 @@ ecma_bigint_parse_string (const lit_utf8_byte_t *string_p, /**< string represena
         digit = (ecma_bigint_digit_t) (character - (LIT_CHAR_LOWERCASE_A - 10));
       }
```

Comment:
Please check what I wrote. The `else if` code should be after this `}`.
So we only check spaces if the next character is not 0-9 and a-f